ospfd/ospf_vty.c

```diff
@@ -8096,8 +8096,8 @@ DEFUN (no_ip_ospf_area,
 	if (argv_find(argv, argc, "(1-65535)", &idx))
 		instance = strtol(argv[idx]->arg, NULL, 10);
 
-	if (ifp->vrf_id && !instance)
-		ospf = ospf_lookup_by_vrf_id(ifp->vrf_id);
+	if (ifp->vrf && !instance)
```

Comment:
Again, how does this not break the test of what is going on here?

Reply:
changed a bit too.